ozaria/site/components/play/PageUnitMap/index.vue

```diff
@@ -39,17 +39,14 @@
       nextLevelOriginal: '',
```

Comment:
This file is mostly formatting. I have pointed out the logic change as a comment.